variables.tf

```diff
@@ -35,7 +35,7 @@ variable "plan_gpu" {
 variable "plan_primary" {
   type        = string
   description = "K8s Primary Plan (Defaults to x86 - baremetal_0)"
```

Comment:
Let's remove the "defaults" part of this comment since it is both redundant (in the Terraform Registry UI) and inaccurate.

Reply:
Fixed.